stubs/reportlab/reportlab/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+from _typeshed import SupportsRichComparison
+from typing import Literal, TypeVar
+
+_SupportsRichComparisonT = TypeVar("_SupportsRichComparisonT", bound=SupportsRichComparison)
+
+Version: str
+__version__: str
+__date__: str
+__min_python_version__: tuple[int, int]
```

Comment:
Nit: These could probably be made `Final`.

Same with other instances of `__version__`.